### ISSUE TITLE
Cerberus FTP 6.x provider doesn't work with Cerberus FTP 12.x

### ISSUE THREAD

**Issue by posixx** [FuseCP/SolidCP#135]
Running Cerberus FTP 12.11.5.0 the SolidCP implementation is broken. No errors are throwed but in the Cerberus eventlog we see:

No SOAP request credentials were provided
Unable to authenticate SOAP request

So the current implementation is broken voor v12 and up.

**Comment by posixx** [comment]
After upgrading CerberusFTP to 12.11.6.0 it started working again so it was a Cerberus problem.

**Comment by posixx** [comment]
Currently using 12.11.6.0 and the FTP provider isn't working. It does not throw any exception (using a wrong admin password it throws an exception) nor does it log a connection attempt within cerberus logs. I contacted Cerberus support, they helped me in setting things up correctly (i was missing the WSDL listener) but after this still not working when trying to view / add accounts. I checked the API using their powershell HelloCerberus.ps1 and i get the correct build number and see the connection in the logs. So the API is working at Cerberus side.
